sound/soc/sof/core.c

```diff
@@ -282,13 +294,17 @@ static int sof_probe(struct platform_device *pdev)
 		goto dbg_err;
 	}
 
+	snd_sof_virtio_miscdev_register(sdev);
```

Comment:
Lets comment why we are registering the misc dev here. Does device need to be always registered ? i.e. the registration should be a kconfig option

Reply:
OK, I will add comments in code. If vBE is enabled, it always needs to be registered.